### PR TITLE
tests: address unaddressed comments from #2064

### PR DESCRIPTION
**Pull request by ianvernon**
Fixes: #2097 

Signed-off by: Ian Vernon <ian@cilium.io>

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: test/helpers/cilium.go [hunk 1]
```diff
@@ -350,9 +350,15 @@ func (s *SSHMeta) PolicyEndpointsSummary() (map[string]int, error) {
...
+	res := s.ListEndpoints()
+	if !res.WasSuccessful() {
+		return nil, fmt.Errorf("was not able to list endpoints: %s", res.CombineOutput().String())
+	}
+
+	endpoints, err := res.Filter("{ [?(@.labels.orchestration-identity[0]!='reserved:health')].policy-enabled }")
+
 	if err != nil {
-		return result, fmt.Errorf("cannot get endpoints")
+		return result, fmt.Errorf("cannot filter for \"policy-enabled\" from output of \"cilium endpoint list\"")
```
Comment (eloycoto) [inline test/helpers/cilium.go]: Why not 
```
fmt.Errorf(`cannot filter for "policy-enabled" from output of "cilium endpoint list"`)
```

It's a bit clear no?

**Review by eloycoto — COMMENTED** [review]
Small nitpick, but feel free to merge ;-)

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Pull request merged**